relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -27,6 +27,25 @@ static TABLE_NAME_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r"(?i)[0-9a-f]{8,
 pub fn parse_query(
     db_system: Option<&str>,
     query: &str,
+) -> Result<Vec<Statement>, sqlparser::parser::ParserError> {
+    match std::panic::catch_unwind(|| parse_query_inner(db_system, query)) {
+        Ok(res) => res,
+        Err(_) => {
```

Comment:
Actually misread the return type. It contains whatever was passed to `panic!()` (almost always some kind of string):

```rs
fn main() {
    if let Err(e) = std::panic::catch_unwind(|| panic!("test")) {
        println!("oops {:?}", e.downcast_ref::<&'static str>());
    }
    
    let r = 123;
    if let Err(e) = std::panic::catch_unwind(|| panic!("oops {r}")) {
        println!("oops {:?}", e.downcast_ref::<String>());
    }
}
```

So in our case it could be useful but most likely it's just a `unwrap()` somewhere.